.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
-sudo: false
 node_js:
-  - "7"
-
+  - 7
 cache:
   yarn: true
-
-install:
-  - yarn
+script:
+  - yarn run test -- --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
The command will only work when run within Travis so I didn't want to add it as a script. I'll add a `yarn coverage` command that makes sense for devs to run locally doe.